packages/lib/src/components/UIElement.tsx

```diff
@@ -145,7 +145,7 @@ export class UIElement<P extends UIElementProps = any> extends BaseElement<P> {
     }
 
     public payButton = props => {
-        return <PayButton {...props} amount={this.props.amount} onClick={this.submit} />;
+        return <PayButton amount={this.props.amount} onClick={this.submit} {...props} />;
```

Comment:
Why have you shifted the position of ...props?
We talked about this once before when I wanted to do this (for BacsDD) and you argued then that it was too fundamental a change and instead the component should override the `payButton` function i.e.
```
 public payButton = props => {
        return <PayButton amount={this.props.amount} onClick={this.submit} {...props} />;
    };
```

Reply:
Good point, moved to the KlarnaPayments element 👍